index.html

```diff
@@ -3356,6 +3356,31 @@ <h3><em>RTL/bidi text (TBD)</em></h3>
 
 
 
+
+
+
+
+<section id="typ_box_posn" class="subtopic">
+<h3>Setting box positioning coordinates when text direction varies</h3>
+
+  	<div class="req" id="vertical_box_posn">
+	<p class="advisement">Box positioning coordinates must take into account whether the text is horizontal or vertical.</p>
```

Comment:
Normative language in BPs should be rendered normatively, e.g. "must take" => "MUST take"

I'm not sure this BP is complete? Also, I note that the paragraph explaining this below talks about RTL vs. LTR presentation. There is no mention of vertical vs. horizontal.